library/core/src/ptr/alignment.rs

```diff
@@ -131,6 +163,92 @@ impl fmt::Debug for Alignment {
     }
 }
 
+#[unstable(feature = "ptr_alignment_type", issue = "102070")]
+impl fmt::Display for Alignment {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        fmt::Display::fmt(&self.as_nonzero(), f)
```

Comment:
Pondering: I have no idea when we should be including `Display` on types.  I guess it's not a problem, but it feels like alignment isn't something I'd usually be showing to people.  But showing it in binary or hex seems logical, so maybe display is ok too.

Reply:
Yeah, that was my thought too. It's less that it's necessarily a common operation, but more that it's well-defined and not a whole lot of extra code to do. I'm fine removing it if it seems too weird.